### PR TITLE
added archived cluster endpoint

### PR DESCRIPTION
**Pull request by pvasant**

### CODE REVIEW

File: model/clusters_mgmt/v1/archived_clusters_resource.model [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
```
Comment (vkareh) [inline model/clusters_mgmt/v1/archived_clusters_resource.model:2]: 2021

File: model/clusters_mgmt/v1/root_resource.model [hunk 2]
```diff
@@ -76,10 +76,14 @@ resource Root {
...
-	
+
 	// Reference to limited support reason templates.
 	locator LimitedSupportReasonTemplates {
 	    target LimitedSupportReasonTemplates
 	}
 
+	locator ArchivedClusters {
+	    target ArchivedClusters
+    }
```
Comment (vkareh) [inline model/clusters_mgmt/v1/root_resource.model:87]: The indentation here looks a bit off

**Comment by vkareh** [comment]
Other than an indentation issue in the root resource, this LGTM

**Comment by pvasant** [comment]
@vkareh  ah i dont know what is the issue with vscode or goland. I use only tabs not spaces still indentation is always off and i had to edit directly to fix it. not sure which editor is the right one to avoid this

**Pull request merged**